pydrive2/auth.py

```diff
@@ -347,6 +348,9 @@ def _InitializeStoragesFromSettings(self):
                     "Please specify credentials file to read"
                 )
             result[backend] = Storage(credentials_file)
+        elif backend == "memory":
+            creds = self.settings.get("save_credentials_json")
+            result[backend] = DictionaryStorage({"creds": creds}, "creds")
```

Comment:
Probably better to `memory` -> `dict` or `dictionary`, since memory is a bit ambiguous.